features/step_definitions/hooks.rb

```diff
@@ -71,13 +71,7 @@
 end
 
 Before '@requires-aruba-version-1' do |scenario|
```

Comment:
Let's remove this all together here.

Reply:
* https://github.com/cucumber/aruba/blob/292ef4cb8faecb4de2d72c15176cbd0d7ad8b5dd/features/04_aruba_api/command/use_last_command_stopped.feature#L58
* https://github.com/cucumber/aruba/blob/292ef4cb8faecb4de2d72c15176cbd0d7ad8b5dd/features/04_aruba_api/command/use_last_command_stopped.feature#L71
* https://github.com/cucumber/aruba/blob/51a034a6380f9686c045cdbb7639fcb922046959/features/03_testing_frameworks/cucumber/steps/command/check_output_of_command.feature#L388

Reply:
I'll fix these.

Reply:
Fixed.